src/utils.js

```diff
@@ -30,4 +30,18 @@ const unquoteString = string => {
   return string;
 };
 
-module.exports = { reduceCSSSelector, unquoteString };
+/**
+ * Semicolon sequences can crash CSSO, so we remove them from CSS text.
+ * github.com/peterbe/minimalcss/issues/243
+ * github.com/css/csso/issues/378
+ * @param {string} css
+ * @return {string}
+ */
+const removeSequentialSemis = css => {
+  while (/;\s*;/.test(css)) {
+    css = css.replace(/;\s*;/g, ';');
+  }
+  return css;
+};
```

Comment:
Now, if you feel like you want to perfect this PR, since that string function got uplifted into its own function, can you add a simple test for it. 

At the moment the `utils.test.js` doesn't exist but it [soon will hopefully](https://github.com/peterbe/minimalcss/pull/257).

Reply:
I _do_ feel like perfecting this PR, but I have a vague suspicion that creating `utils.test.js` in two separate branches might introduce some kind of merge conflict.

Reply:
If you rebase your branch against origin/master you should have a file now that you can add to.

Reply:
Rebase without squash that is.

Reply:
That didn't go as planned...
I think it's going to be simpler for me to submit another pull request, replacing this one.
Squashing into https://github.com/peterbe/minimalcss/compare/master...jc275:remove-sequential-semicolons